ui/raidboss/data/02-arr/raid/t1.ts

```diff
@@ -30,8 +31,9 @@ const triggerSet: TriggerSet<Data> = {
       response: Responses.interrupt(),
     },
     {
-      // Indiscriminate Hood Swing
       id: 'T1 Initiated',
+      // Indiscriminate Hood Swing
```

Comment:
```suggestion
      // Indiscriminate Hood Swing
      id: 'T1 Initiated',
```

Reply:
Don't quote me on this, but I think all of these comments ended up after `id` because I used the automatic-order-fix option? Something to look into in the future I guess?